quick_start.rst

```diff
@@ -216,6 +216,30 @@ supported by {Project}. To see a comparison of the {Project}
 definition file with Dockerfile, please see: :ref:`this section
 <sec:deffile-vs-dockerfile>`.
 
+.. note::
+
+   If you are using {Project} through `Lima <https://lima-vm.io>`_,
```

Comment:
Refer to the admin doc section here instead of the Lima page.  I think it makes more sense to put project links in the admin doc.  There is already one there pointing to the github of lima but maybe that could be improved.